server/graphql/v1/mutations/orders.js

```diff
@@ -960,3 +965,41 @@ export async function addFundsToCollective(order, remoteUser) {
 
   return models.Order.findByPk(orderCreated.id);
 }
+
+export async function createOrderForDispatch(order, loaders, remoteUser, reqIp) {
```

Comment:
- the mutation should be named `dispatchOrder`
- it should take as argument one **already processed** order
- and should return a list of the generated Orders, with the proper GraphQL type